### ISSUE TITLE
jetson nano no CUDA-capable device is detected

### ISSUE THREAD

**Issue by haiyang-tju** [idavis/jetson-containers#17]
Thanks for the great work！I followed this project and build the `Custom Root Filesystems` and flashed the device.

Everything goes ok, and the device can start normally.

Next step, I have two ways to start a cuda-based program：
- Install cuda-toolkit directly on the device and compile and run it.
- Install cuda-toolkit in `Custom Root Filesystems` and flash it to the device.

Unfortunately, neither option is feasible. I am stucked on this ERRORS:
```
./deviceQuery 
./deviceQuery Starting...

 CUDA Device Query (Runtime API) version (CUDART static linking)

cudaGetDeviceCount returned 38
-> no CUDA-capable device is detected
Result = FAIL
```

Did it an nvidia cuda driver problem？

I am on the jetson nano device, and build the `Custom Root Filesystems` with `ubuntu-base-18.04.1-base-arm64` under jetson4.2.

**Comment by haiyang-tju** [comment]
The steps I followed are there：
- tar the `DRIVER_PACK Jetson-210_Linux_R32.1.0_aarch64.tbz2`.
- and also tar the `Custom Root Filesystems` into the directory `/Linux_for_Tegra/rootfs`, I used `chroot` to build my custom applications, such as `jetson-fan-ctl` to control the device fan boot automatically.
- run the script `apply_binaries.sh` and it gives the outputs `Success! Package Install Finished Successfully`.
- finally flash the device by `sudo ./flash.sh -S 29496MiB -e 29504MiB jetson-nano-qspi-sd mmcblk0p1` with my 32GB tf card.

The device start normally and the fan runs automatically. 

- next, I installed the cuda-toolkit with the deb package `cuda-repo-l4t-10-0-local-10.0.166_1.0-1_arm64.deb`. 
- then, I compiled the cuda sample program `deviceQuery` with `sudo make`, and generated the run file. BUT the program gives the ERROR：`no CUDA-capable device is detected`.
